Hedvig/Generic/Viewables/CircleLabelWithSubLabel.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CircleLabelWithSubLabel.swift
+//  Hedvig
+//
+//  Created by Sam Pettersson on 2019-01-04.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Foundation
+import UIKit
+
+struct CircleLabelWithSubLabel {
+    let labelText: DynamicString
+    let subLabelText: DynamicString
+    let color: String
+
+    init(labelText: DynamicString, subLabelText: DynamicString, color: String) {
+        self.labelText = labelText
+        self.subLabelText = subLabelText
+        self.color = color
+    }
+}
+
+extension CircleLabelWithSubLabel: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let circleView = UIView()
+        let bag = DisposeBag()
+
+        let labelsContainer = CenterAllStackView()
+        labelsContainer.axis = .vertical
+
+        let titleLabel = UILabel()
+        titleLabel.font = HedvigFonts.circularStdBold?.withSize(48)
+        titleLabel.lineBreakMode = .byWordWrapping
+        titleLabel.numberOfLines = 0
+        titleLabel.textAlignment = .center
+        titleLabel.adjustsFontSizeToFitWidth = true
+        bag += titleLabel.setDynamicText(labelText)
+
+        let subLabel = UILabel()
+        subLabel.font = HedvigFonts.circularStdBook?.withSize(18)
+        subLabel.lineBreakMode = .byWordWrapping
+        subLabel.numberOfLines = 0
+        subLabel.textAlignment = .center
+        subLabel.adjustsFontSizeToFitWidth = true
+        bag += subLabel.setDynamicText(subLabelText)
+
+        circleView.layer.shadowOpacity = 0.2
```

Comment:
hmm, it's not that nice to have the same shadow specification scattered around, would be nice if we could keep them in one place 🤔

Reply:
@Iamsamwhoami I agree. Will fix 👍